Server/mods/deathmatch/logic/CResource.cpp

```diff
@@ -1300,6 +1314,23 @@ bool CResource::GetFilePath(const char* szFilename, string& strPath)
     return FileExists(strPath);
 }
 
+std::vector<std::string> CResource::GetFilePaths(const char* szFilename)
+{
+    std::vector<std::string>    vecFiles;
+    std::string&                strDirectory = IsResourceZip() ? m_strResourceCachePath : m_strResourceDirectoryPath;
```

Comment:
Should be `const std::string&` because you obviously don't want to change those class fields.